### PR TITLE
ENH: Adding new feature in main_window.py that allows user to switch …

### PR DESCRIPTION
**Pull request by c-tsoi**
…between stylesheets in an active pydm window.

Per request from scientists, I have implemented a stylesheet switcher that allows a user to change their stylesheet in an active PyDM window. They are able to change it multiple times with any qss or css file. I have done some basic testing with the UI in an active session but have not done any unit tests on this.

### CODE REVIEW

**Comment by jbellister-slac** [comment]
Thanks! We'll take a look at this soon

File: pydm/pydm_ui.py [hunk 1]
```diff
@@ -189,3 +190,17 @@ def retranslateUi(self, MainWindow):
         self.actionDefault_Font_Size.setText(_translate("MainWindow", "Default Font Size"))
         self.actionDefault_Font_Size.setShortcut(_translate("MainWindow", "Ctrl+0"))
         self.actionQuit.setText(_translate("MainWindow", "Quit"))
+        self.actionSave.setText(_translate("MainWindow", "Save"))
+        self.actionSave_As.setText(_translate("MainWindow", "Save As..."))
+        self.actionLoad.setText(_translate("MainWindow", "Load"))
+        self.actionChange_Stylesheet.setText(_translate("MainWindow", "Change Stylesheet"))
+
+
+if __name__ == "__main__":
```
Comment (YektaY) [inline pydm/pydm_ui.py:199]: I think this if statement could be removed, which would keep the code more succinct

File: pydm/pydm_ui.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 # WARNING! All changes made in this file will be lost!
 
 
-from qtpy import QtCore, QtGui, QtWidgets
+from PyQt5 import QtCore, QtGui, QtWidgets
```
Comment (YektaY) [inline pydm/pydm_ui.py:10]: I would recommend reverting this back to qtpy to keep in line with the rest of pydm

**Review by YektaY — COMMENTED** [review]
I left a couple comments on small recommended changes to the PR. Otherwise the functionality works great for me and I am sure the users would be very happy with this additional functionality. Thank you

**Comment by YektaY** [comment]
this looks great, thank you for the feature!

**Pull request merged**